Gemfile

```diff
@@ -103,6 +104,8 @@ group :test do
   gem 'capybara'
   gem 'factory_bot_rails'
   gem 'rspec-rails'
+  gem 'rubocop-capybara'
```

Comment:
You have Gemfile changes for this but they should be in the same block. Put only those gems you need in the shared `dev/test` block. If a gem is already coming in because something else depends on it, you needn't require it.